src/sinks/datadog/events/config.rs

```diff
@@ -90,8 +93,14 @@ impl DatadogEventsConfig {
 impl SinkConfig for DatadogEventsConfig {
     async fn build(&self, cx: SinkContext) -> crate::Result<(VectorSink, Healthcheck)> {
         let client = self.build_client(cx.proxy())?;
-        let healthcheck = self.dd_common.build_healthcheck(client.clone())?;
-        let sink = self.build_sink(client)?;
+        let global = cx
+            .extra_context
+            .get::<datadog::Options>()
+            .cloned()
+            .unwrap_or_default();
```

Comment:
Nice! This is really clean. I love that we can easily default to the environment variables here. 😄